pkg/rslog/README.md

```diff
@@ -2,10 +2,35 @@
 
 ## Description
 
-A logging library that meets the requirements of the [Posit logging standards](
-https://rstudiopbc.atlassian.net/wiki/spaces/ENG/pages/36048281/RStudio+Logging+Standard).
+A logging library that meets the requirements of the Posit logging standards.
 Provides a logger that wraps `github.com/sirupsen/logrus`.
 
+### The Posit logging standards
+
+Excerpts here come from the internal confluence space "RStudio Logging Standard"
+
+* Configuration via environment variables (that override values in the configuration file) is encouraged.
+* The output of logging should be file/syslog or stderr.  You can also support logging to file and syslog separately.
+  Kubernetes expects logging to stdout/stderr and can have limited local storage that can be broken by file logging.
+* File/syslog logging must always be supported and be the default type. Files should go in a location that is shared
+  by all products.
+* Output to stderr must be supported, including from child processes where possible. Program output where needed will
+  go to stdout, while log messages will go to stderr.
+* There should be a single log file per service-child process logs should be read into the parent to be put into the
```

Comment:
> There should be a single log file per service-child process logs should be read into the parent to be put into the...

Should this be two sentences?

Reply:
Yeah... clumsy editing. Thanks for catching that.